src/main/java/gregtech/common/metatileentities/electric/multiblockpart/MetaTileEntityMultiblockPart.java

```diff
@@ -158,6 +159,18 @@ public void addToMultiBlock(MultiblockControllerBase controllerBase) {
     @Override
     public void removeFromMultiBlock(MultiblockControllerBase controllerBase) {
         setController(null);
+        if (this.importItems instanceof INotifiableHandler) {
+            ((INotifiableHandler) importItems).removeNotifiableMetaTileEntity(controllerBase);
+        }
+        if (this.exportItems instanceof INotifiableHandler) {
+            ((INotifiableHandler) exportItems).removeNotifiableMetaTileEntity(controllerBase);
+        }
+        if (this.importFluids instanceof INotifiableHandler) {
+            ((INotifiableHandler) fluidInventory).removeNotifiableMetaTileEntity(controllerBase);
+        }
+        if (this.exportFluids instanceof INotifiableHandler) {
+            ((INotifiableHandler) fluidInventory).removeNotifiableMetaTileEntity(controllerBase);
```

Comment:
Do we want to be removing the fluid inputs and outputs from `fluidInventory` or from `notifiedFluidInputList` and `notifiedFluidOutputList`?